packages/integration-karma/scripts/karma-plugins/hydration-tests.js

```diff
@@ -63,6 +63,13 @@ async function getCompiledModule(dirName) {
         cache,
 
         external: ['lwc', 'test-utils', '@test/loader'], // @todo: add ssr modules for test-utils and @test/loader
+
+        onwarn(warning) {
+            // Ignore warnings from our own Rollup plugin
+            if (warning.plugin !== 'rollup-plugin-lwc-compiler') {
+                console.warn(warning.message);
+            }
+        },
```

Comment:
The `onwarn` handler does expose a method to pretty-print the rollup warning. More details about this here: https://rollupjs.org/guide/en/#onwarn

```suggestion
        onwarn(warning, warn) {
            // Ignore warnings from our own Rollup plugin
            if (warning.plugin !== 'rollup-plugin-lwc-compiler') return;

            warn(warning);
        },
```

Reply:
Thank you, TIL!

Reply:
Fixed.